geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/z2/XZ2WritableIndex.scala

```diff
@@ -20,29 +20,31 @@ import org.locationtech.geomesa.accumulo.AccumuloVersion
 import org.locationtech.geomesa.accumulo.data._
 import org.locationtech.geomesa.accumulo.data.tables.GeoMesaTable
 import org.locationtech.geomesa.accumulo.index.AccumuloWritableIndex
+import org.locationtech.geomesa.curve.XZ2SFC
 import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
 import org.opengis.feature.simple.SimpleFeatureType
 
 trait XZ2WritableIndex extends AccumuloWritableIndex {
 
   import AccumuloWritableIndex.{DefaultNumSplits, DefaultSplitArrays}
-  import XZ2Index.SFC
 
   override def writer(sft: SimpleFeatureType, ops: AccumuloDataStore): (WritableFeature) => Seq[Mutation] = {
+    val sfc = XZ2SFC(sft.getXZPrecision)
```

Comment:
Could we still have a lazy variable at the trait level that gets shared among these routines?